stable/insights-agent/templates/opa/rbac.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.opa.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-opa
+  labels:
+    app: insights-agent
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-checks-view
+  labels:
+    app: insights-agent
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+rules:
+- apiGroups:
+  - 'insights.fairwinds.com'
+  resources:
+  - 'customchecks'
+  - 'customcheckinstances'
+  verbs:
+  - 'get'
+  - 'list'
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-opa
+  labels:
+    app: insights-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "insights-agent.fullname" . }}-opa
+    namespace: {{ .Release.Namespace }}
+{{- end -}}
```

Comment:
we should add the ability to add additional cluster roles for this ServiceAccount, in case they want to check a resource that hasn't been aggregated to cluster-view

Reply:
Yeah that's probably a good idea. I always struggle with that. The right way to do it seems to be to aggregate into view (like we're doing here), but not everyone seems to follow that. Also they could legitimately want to run an opa check on secrets for some reason.